metagraph/src/cli/build.cpp

```diff
@@ -140,6 +153,7 @@ int build_graph(Config *config) {
             } else {
                 graph_data.reset(next_chunk);
             }
+            checkpoint.done();
```

Comment:
It's not clear what this method does when I look at its name. Maybe cleanup? Then call it `.remove_checkpoint()`